cli/smartcontract/smart_contract.go

```diff
@@ -150,6 +150,10 @@ func NewCommands() []cli.Command {
 						Name:  "config, c",
 						Usage: "Configuration input file (*.yml)",
 					},
+					cli.BoolFlag{
+						Name:  "check-standards",
```

Comment:
I think it should be the default behavior, this switch can turn it off if needed.

Reply:
Fixed.